src/components/Notification/Notification.svelte

```diff
@@ -4,20 +4,23 @@
 	import Icons from 'components/icons';
 
 	const remove = (id: string) => () => {
-		notifications.update(_notifications => _notifications.filter(notification => notification.id !== id));
```

Comment:
**suggestion** : use the notifications as a dictionnary, not an array
instead of : `[{ id: 1, message: "notif1"}, {id: 2, message: "notif 2"}]`
you can do : `{ 1: {message: "notif1"}, 2: {message: "notif2"} }`

you could remove all the find and retrieve the notification with `notifications[id]`

(but you will have trouble if the order of the notifications is important)